packages/amplication-data-service-generator/src/server/prisma/constants.ts

```diff
@@ -1,13 +1,13 @@
-import { PrismaDataSource } from "@amplication/code-gen-types";
+import { PrismaDataSource, PrismaClientGenerator } from "@amplication/code-gen-types";
 import * as PrismaSchemaDSL from "prisma-schema-dsl";
```

Comment:
import should be removed

Reply:
which import? I need all of them
![Screen Shot 2022-09-17 at 13 50 48](https://user-images.githubusercontent.com/39680385/190853112-1ae9cba3-f4c5-4364-9fd0-c8806c9f7cef.png)